mongodb-quickstart/src/main/java/org/acme/mongodb/CodecFruitService.java

```diff
@@ -35,6 +35,6 @@ public void add(Fruit fruit) {
     }
 
     private MongoCollection<Fruit> getCollection() {
-        return mongoClient.getDatabase("fruit").getCollection("fruit", Fruit.class);
+        return mongoClient.getDatabase("fruit").getCollection("codec_fruit", Fruit.class);
```

Comment:
Why did you change the collection name?

Reply:
To make it easier to run the same test on 3 different resources, each one of them using slightly different approach (classic, with codec, reactive).

Reply:
Ah ok! Thanks for the clarification